docs/proposals/capacity-planning-refactoring.md

```diff
@@ -0,0 +1,100 @@
+# Proposal: Refactoring of capacity planning
+
+Last updated: 09.08.2021
+
+
+## Abstract
+
+Refactor logic of AC reservation to consider cases for 2 or more LVG PVs
+
+## Background
+
+Currently, 2 or more LVG PVs can be assigned to different disks on the reservation phase. It leads to the ineffective space usage.
+
+Causes:
+1. Reservation Controller doesn't consider on the planning phase, that LVG volumes should use one device. 
+   If ACR reserves AC for LVG, other ACR won't be able to use the same drive for LVG.
+
+2. CSI reserves capacity for pvc of one pod on all nodes. After volumes creation, reservation will be released. 
+   ACR controller selects drives with the closest capacity for the first reservation. 
+   But if there are some existed reservation, they will affect each other.
+
+   ![Screenshot](images/ACR_limitations.png)
```

Comment:
Picture is unclear. Please add information about what these colors mean.

Reply:
updated